linera-service/tests/common/mod.rs

```diff
@@ -1,8 +1,9 @@
 // Copyright (c) Zefchain Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-use once_cell::sync::Lazy;
-use tokio::sync::Mutex;
+#[cfg(any(feature = "aws", feature = "rocksdb", feature = "scylladb"))]
+use {once_cell::sync::Lazy, tokio::sync::Mutex};
 
 /// A static lock to prevent integration tests from running in parallel.
+#[cfg(any(feature = "aws", feature = "rocksdb", feature = "scylladb"))]
```

Comment:
Here and above, this is not needed because the module is already tagged.

Reply:
Well, it was needed for passing the CI.